BaseFilter/sections/antiadblock.txt

```diff
@@ -2546,10 +2546,9 @@ videosgay.me#%#//scriptlet('set-constant', 'cRAds', 'true')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/163449
 perchance.org#%#//scriptlet('set-constant', 'ad1sAreShowing', 'trueFunc')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/163466
-smsonline.cloud#@##ad-slot
-smsonline.cloud#@#ins.adsbygoogle[data-ad-client]
-smsonline.cloud#@#ins.adsbygoogle[data-ad-slot]
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=smsonline.cloud
+smsonline.cloud#%#AG_onLoad(function(){function a(){document.cookie.split(";").forEach(c=>{const n=c.split("=")[0].trim();document.cookie=`${n}=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;`});}function b(){a();location.reload();}function init(){const e=document.querySelector('.reloadContainer + div[style]');if(e&&e.querySelector('button[type="button"]'))e.addEventListener('click',b);}init();window.reloadAndClearCookies=b;})();

```

Comment:
As far as I understand, it clears cookies and reload website when `Reload messages` button is clicked (I guess that this part `window.reloadAndClearCookies=b;` can be removed), I think that it's fine as a workaround, but maybe we could try to add a rule which will fix it without reloading website?

For example something like this:
```adblock
smsonline.cloud#%#(()=>{const e=()=>{document.querySelectorAll(".chakra-portal").forEach((e=>{e.querySelector('.chakra-modal__overlay[style*="opacity"]')&&e.setAttribute("style","display: none !important;")}))},t=()=>{},a=function(t,a){const r={name:t,listener:a};requestAnimationFrame((()=>{try{"rewardedSlotGranted"===r.name&&setTimeout(e,2e3),r.listener()}catch(e){}}))};window.googletag={cmd:[],pubads:()=>({addEventListener:a,removeEventListener:t,refresh:t,getTargeting:()=>[],setTargeting:t,disableInitialLoad:t,enableSingleRequest:t,collapseEmptyDivs:t,getSlots:t}),defineSlot:()=>({addService(){}}),defineOutOfPageSlot:t,enableServices:t,display:t,enums:{OutOfPageFormat:{REWARDED:1}}},googletag.cmd.push=e=>{try{e()}catch(e){}return 1}})();
```

Reply:
Thanks, changed it, looks more appropriate